workflow/storage/cosmosdb/cosmosdb.go

```
@@ -216,7 +205,7 @@ func (v *Vault) createContainerClient(ctx context.Context) (*azcosmos.ContainerC
 func (v *Vault) createContainer(ctx context.Context, database *azcosmos.DatabaseClient, indexPaths []azcosmos.IncludedPath) (string, error) {
 	v.props.ID = v.container
 	v.props.PartitionKeyDefinition = azcosmos.PartitionKeyDefinition{
-		Paths: []string{"/pk"},
+		Paths: []string{"/partitionKey"},
```

while I doubt it matters much in the scheme of things, why make this key (which will be in every document) longer? is pk too vague?

---

Just seemed clearer and everything else seemed to be in that vein.  Orignally, I changed it all to planID.  

Neet feature, if your partition key on the call is different that the defined partition key, it silently fails.